### PR TITLE
AddProfile

### PR DESCRIPTION
**Pull request by irfanariyaz**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1879"></a>

### CODE REVIEW

File: public/data/irfanariyaz.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "bio": "Full-Stack developer starting to opensource",
+  "avatar": "https://github.com/irfanariyaz.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/irfanariyaz",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Linkedin",
```
Comment (Panquesito7) [inline public/data/irfanariyaz.json:13]: I recommend capitalizing the `i` here. What do you think?
```suggestion
      "name": "Follow me on LinkedIn",
```

**Review by EmmaDawsonDev — APPROVED** [review]

File: public/data/irfanariyaz.json (hunk 1 above)
Comment (irfanariyaz) [reply public/data/irfanariyaz.json]: that would be great.. do I need to do a pull request after the changes are made??

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by irfanariyaz** [comment]
I cannot see my .json file in the EddieHubCommunity:main? do i need to do a pull request again?

**Comment by Panquesito7** [comment]
It might take a little while to deploy or show up. 🙂

**Comment by irfanariyaz** [comment]
ok thanks .How do I merge my main  and  AddProfile branch ?

**Comment by irfanariyaz** [comment]
Thanks, I can see it now and deployed it using Gitpod. Saw my profile there too. Excited to do more open source projects. Big shout out to Eddiehub community.

**Pull request merged**